src/pages/discover.js

```diff
@@ -106,9 +106,13 @@ function useCollectives(query) {
   return state;
 }
 
-const DiscoverPage = ({ router }) => {
-  const { query } = router;
-  const { collectives, offset, total, show, tags, collective = [] } = useCollectives(query);
+const DiscoverPage = ({ all, router, ...props }) => {
+  const {
+    query: { show },
+  } = router;
```

Comment:
@Betree When I added this I got no errors though ...but I had a slow network so I couldn't load all the cards